DQM/BeamMonitor/plugins/Vx3DHLTAnalyzer.cc

```diff
@@ -569,14 +563,12 @@ int Vx3DHLTAnalyzer::MyFit(vector<double>* vals) {
           break;
         }
     if (goodData == 0) {
```

Comment:
I don't think this is correct. Here should be as before, because after calling Gaus3D->Minimize()
Gaus3D->X() will return a pointer to the minimum X values found , while vals() will contain the values before minimisation